### PR TITLE
Adding ability to add and remove moderators cleanly in the admin app. [ENG-2372]

### PR DESCRIPTION
**Pull request by corbinSanders**
## Purpose

Adding the ability to add admin moderators cleanly to registration providers in the admin app.

## Changes

Modifying templates. Adding code to the views. 

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify you can add and remove admins to registration providers
- Verify you can remove a moderator as an admin, but not as a moderator

What are the areas of risk?

Any concerns/considerations/questions that development raised? N/A

## Documentation

N/A

## Side Effects

Shouldn't be.
## Ticket

https://openscience.atlassian.net/browse/ENG-2372

### CODE REVIEW

File: admin/templates/registration_providers/edit_moderators.html [hunk 1]
```diff
@@ -22,30 +22,40 @@ <h2>{{ registration_provider.name }}</h2>
...
+                <form id="remove-moderators-form" action="{% url 'registration_providers:remove_admins_and_moderators' registration_provider.id %}" method="post">
                     {% csrf_token %}
                     <table class="table table-striped">
                         <th></th>
                         <th>Name</th>
+                        <th>Type</th>
                         {% for moderator in moderators %}
                             <tr>
-                                <td><input type='checkbox' name="{{moderator.id}}"></td>
+                                <td><input type='checkbox' name="Moderator-{{moderator.id}}"></td>
```
Comment (Johnetordoff) [inline admin/templates/registration_providers/edit_moderators.html:45]: Can you just make this a non-name HTTP tag attribute and have the that be the thing that distinguishes them? Like `<input type='checkbox' name="{{moderator.id}}" group="moderator"></td>` ?

**Review by Johnetordoff — COMMENTED** [review]
Looks good, just one question.

File: admin/templates/registration_providers/edit_moderators.html (hunk 1 above)
Comment (corbinSanders) [reply admin/templates/registration_providers/edit_moderators.html:45]: It looks like the group attribute isn't passed off in the request.

**Review by brianjgeiger — APPROVED** [review]

**Comment by brianjgeiger** [comment]
This is good, with the exception of the failing tests.

```
==================================== ERRORS ====================================
3873________________ ERROR at setup of TestEditModerators.test_get _________________
3874admin_tests/registration_providers/test_views.py:283: in add_moderator_view
3875    view = views.AddModerators()
3876E   AttributeError: module 'admin.registration_providers.views' has no attribute 'AddModerators'
3877____________ ERROR at setup of TestEditModerators.test_post_remove _____________
3878admin_tests/registration_providers/test_views.py:276: in remove_moderator_view
3879    view = views.RemoveModerators()
3880E   AttributeError: module 'admin.registration_providers.views' has no attribute 'RemoveModerators'
3881______________ ERROR at setup of TestEditModerators.test_post_add ______________
3882admin_tests/registration_providers/test_views.py:283: in add_moderator_view
3883    view = views.AddModerators()
3884E   AttributeError: module 'admin.registration_providers.views' has no attribute 'AddModerators'
```

**Pull request merged**